cmd/sovereignnode/config/sovereignConfig.toml

```diff
@@ -30,6 +30,10 @@
     MainChainNotarizationStartRound = 11
 
 [OutgoingSubscribedEvents]
+    # Time to wait in seconds for outgoing operations that need to be bridged from sovereign chain to main chain.
+    # If no confirmation of bridged data is received after this time, next leader should retry sending data.
+    TimeToWaitForUnconfirmedOutGoingOperation = 30
```

Comment:
It is already specified in the comment that it is in seconds

Reply:
it would be helpful to be direct in the name, not to look in the toml file. 
in the sovereignConfig.go there is no comment and no mention that the time is in seconds

Reply:
Added `InSeconds`